src/Streamly/Internal/Data/Stream/IsStream/Type.hs

```diff
@@ -463,9 +464,9 @@ foldStream st yld sng stp m =
 fromSerial :: IsStream t => SerialT m a -> t m a
 fromSerial = adapt
 
-instance IsStream SerialT where
-    toStream = getSerialT
-    fromStream = SerialT
+instance IsStream Stream.Stream where
```

Comment:
This should be in Stream module

Reply:
The instance is with the type class definition. And this has to be in streamly module cannot be in streamly-core.